.env.development

```diff
@@ -17,3 +17,4 @@ SEGMENT_KEY=null
 SITE_NAME='edX'
 USER_INFO_COOKIE_NAME='edx-user-info'
 BLOCKSTORE_COLLECTION_UUID=null
+SECURE_ORIGIN_XBLOCK_BOOTSTRAP_HTML_URL=/xblock-bootstrap.html
```

Comment:
Nit: Could we place this setting above `BLOCKSTORE_COLLECTION_UUID` since it's placed at the bottom for convenience and all devs will need to edit it?

Reply:
Good point.  Will do, thanks!